src/openagi/actions/tools/arxiv_search.py

```diff
@@ -0,0 +1,72 @@
+from openagi.actions.base import BaseAction
+from pydantic import Field
+from openagi.exception import OpenAGIException
+from typing import ClassVar, Dict, Any
+
+try:
+    import arxiv
```

Comment:
If we import this one will it work or explicitly we will have to run the install command? @tarun-aiplanet

Reply:
we will have to run install command